### PR TITLE
6607: fixed scrolling issue to latest message

### PR DESCRIPTION
**Pull request by nikola-gladovic**
#6607 

I think it is unused code.

### CODE REVIEW

**Review by marcoambrosini — COMMENTED** [review]
Why unused? The content of that watcher runs every time the tab is focused in theory

**Comment by marcoambrosini** [comment]
@PVince81 do you remember what was the reasoning behind scrolling to the first unread every time a tab is refocused btw?

**Comment by PVince81** [comment]
> @PVince81 do you remember what was the reasoning behind scrolling to the first unread every time a tab is refocused btw?

not exactly.
probably was a UX thing so that when you return to the hidden tab you start at the first unread message
an improvement could be to add a link somewhere "jump to the first unread message"

**Comment by marcoambrosini** [comment]
yes we could have that, so let's remove this then

File: src/components/MessagesList/MessagesList.vue [hunk 1]
```diff
@@ -238,14 +238,14 @@ export default {
...
-				// need to investigate why the sidebar takes that long to open and is not even animated
-				window.setTimeout(() => this.scrollToFocussedMessage(), 100)
-			}
-		},
+		// isWindowVisible(visible) {
+		// 	if (visible) {
+		// 		this.onWindowFocus()
+		// 		// FIXME: the sidebar chat takes much longer to open, this is why we need a higher value here
+		// 		// need to investigate why the sidebar takes that long to open and is not even animated
+		// 		window.setTimeout(() => this.scrollToFocussedMessage(), 100)
```
Comment (marcoambrosini) [inline src/components/MessagesList/MessagesList.vue:246]: This is the part we need to remove @nikola-gladovic

File: src/components/MessagesList/MessagesList.vue [hunk 2]
```diff
@@ -238,14 +238,14 @@ export default {
...
-			if (visible) {
-				this.onWindowFocus()
-				// FIXME: the sidebar chat takes much longer to open, this is why we need a higher value here
-				// need to investigate why the sidebar takes that long to open and is not even animated
-				window.setTimeout(() => this.scrollToFocussedMessage(), 100)
-			}
-		},
+		// isWindowVisible(visible) {
+		// 	if (visible) {
+		// 		this.onWindowFocus()
```
Comment (marcoambrosini) [inline src/components/MessagesList/MessagesList.vue:243]: This we should keep

File: src/components/MessagesList/MessagesList.vue (hunk 1 above)
Comment (nikola-gladovic) [reply src/components/MessagesList/MessagesList.vue:246]: Ok, sure

File: src/components/MessagesList/MessagesList.vue (hunk 2 above)
Comment (nikola-gladovic) [reply src/components/MessagesList/MessagesList.vue:243]: Ok

**Review by marcoambrosini — COMMENTED** [review]
Can you rebase so that there is only one commit please? Looking good otherwise

**Review by marcoambrosini — APPROVED** [review]

**Pull request merged**